apps/documentation/src/routes/documentation/index.mdx

```diff
@@ -1,9 +1,10 @@
-import { Head } from 'tuono'
+import MetaTags from '../../components/meta-tags'
```

Comment:
**[LanguageTool]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>This sentence does not start with an uppercase letter. (UPPERCASE_SENTENCE_START)
Suggestions: `Import`
URL: https://languagetool.org/insights/post/spelling-capital-letters/ 
Rule: https://community.languagetool.org/rule/show/UPPERCASE_SENTENCE_START?lang=en-US
Category: CASING
<!-- __reviewdog__:ChA5NzhmZGE3ODZlMDUxNDFlEgxMYW5ndWFnZVRvb2w= -->